plugins/event_notification/providers/bpm/lib/api/filters/KalturaBusinessProcessServerFilter.php

```diff
@@ -6,12 +6,12 @@
 class KalturaBusinessProcessServerFilter extends KalturaBusinessProcessServerBaseFilter
 {
 	/**
-	 * @var KalturaNullableBoolean
+	 * @var bool
```

Comment:
Boolean can't be null and therefor it will be false if it wasn't supplied, I think that using KalturaNullableBoolean was better.